components/splitter/panes.md

```diff
@@ -6,59 +6,301 @@ slug: splitter-panes
         </SplitterPane>
 
-        <SplitterPane Collapsible="false" >
+        <SplitterPane Collapsible="false" Class="k-scrollable">
             <h4>Right Pane</h4>
-            <span>Non-collapsible pane. No size set, it will take the remaining space of the component.</span>
+            <div style="height:150%">Non-collapsible and scrollable pane with no size. It will take up the remaining space in the component.</div>
         </SplitterPane>
 
-        <SplitterPane Visible="false" >
-            <h4>Not Visible Pane</h4>
-            <span>This pane is not displayed.</span>
+        <SplitterPane Visible="false">
+            <h4>Invisible Pane</h4>
+            <span>This pane will not render.</span>
         </SplitterPane>
-        
+
     </SplitterPanes>
 </TelerikSplitter>
+
+@code {
+    private string PaneSize1 { get; set; } = "200px";
+
+    private string PaneSize2 { get; set; } = "20%";
+}
 ````
 
+## Pane Scrolling
+
+To make a Splitter pane scrollable without using additional HTML markup or custom CSS styles, set the pane `Class` parameter to `k-scrollable`. See the example above.
+
+## Pane Dimensions
+
+The dimensions of a Splitter pane depend on:
+
+* The [pane `Size`](#pane-size) parameter
+* The [pane `Collapsible` and `Resizable`](#pane-collapsibility-and-resizability) parameters
+* The [Splitter `Width`, `Height`, and `Orientation`](#splitter-width-and-height) parameters
+
+The sections below provide more details and a [hands-on example](#example).
+
+### Pane Size
+
+The Splitter pane `Size` controls the pane width or height, depending on the [Splitter `Orientation`]({%slug splitter-orientation%}).
+
+There must be at least one `SplitterPane` without a `Size`. This pane will adjust automatically to occupy the remaining space, based on the other pane sizes.
+
+If the pane `Size` is greater than `Max`, the pane cannot be resized even if its `Resizable` parameter is set to `true`.
+
+### Pane Collapsibility and Resizability
+
+Collapsibility and resizability have the following impact on the Splitter pane dimensions:
+
+* Panes that are collapsible or resizable are called *flex panes*. When a flex pane has no `Size`, it expands to fill the available space. If multiple flex panes have no `Size`, they take up equal parts of the available space.
+* Panes that are not collapsible and not resizable are called *static panes*. When a static pane has no `Size`, it expand and shrinks based on its content.
+
+### Splitter Width and Height
+
+In a [vertical Splitter]({%slug splitter-orientation%}), the pane widths match the Splitter `Width`.
+
+Here is how the Splitter `Height` affects the pane heights:
+
+* If a [horizontal Splitter]({%slug splitter-orientation%}) has no `Height`, then its panes do not expand vertically to fill up the Splitter element. The [example](#example) below shows how to workaround this with a `height:auto` style on the `.k-pane` class.
```

Comment:
Usually, `workaround` is the noun or adjective form, while `work around` would be the verb.